app/facades/item_facade.rb

```diff
@@ -0,0 +1,29 @@
+class ItemFacade
+
+  def self.all_items
+    all_items = items_data[:data].map do |data|
+      ItemPoro.new(data)
+    end
+  end
+
+  def self.find_items(merchant_id)
+    items = all_items.select do |i|
+      i.merchant_id.to_s == merchant_id
+    end
+  end
+
+  def self.find_item(item_id)
+    all_items.find do |i|
+      i.id == item_id
+    end
```

Comment:
Similar to above, I'd add an endpoint for `api/v1/items/{:id}` to get the items back directly instead of iterating through all items to find a match.